src/main/java/duke/commands/CommandManager.java

```diff
@@ -93,9 +93,8 @@ public static Command manageCommand(String userInput) throws DukeException {
             try {
                 return new DeleteAssignedTaskCommand(parser.parseUserInput()[0]);
             } catch (Exception e) {
-                throw new DukeException(CommandManager.class, "Please follow the `delete assigned task :#<patient id>"
-                        + " :#<task id>` or "
```

Comment:
This "CommandManager.class" is added for the logger. So do call this method instead.

Reply:
I had to change the message itself due to its format being incorrect, but the CommandManager.class parameter has been added back in.